jetty-ee9/jetty-ee9-bom/pom.xml

```diff
@@ -128,7 +125,21 @@
         <artifactId>jetty-ee9-webapp</artifactId>
         <version>12.0.0-SNAPSHOT</version>
       </dependency>
-
+      <dependency>
+        <groupId>org.eclipse.jetty.ee9.osgi</groupId>
+        <artifactId>jetty-ee9-osgi-boot</artifactId>
+        <version>12.0.0-SNAPSHOT</version>
+      </dependency>
+      <dependency>
+        <groupId>org.eclipse.jetty.ee9.osgi</groupId>
+        <artifactId>jetty-ee9-osgi-boot-jsp</artifactId>
+        <version>12.0.0-SNAPSHOT</version>
+      </dependency>
+      <dependency>
+        <groupId>org.eclipse.jetty.ee9.osgi</groupId>
+        <artifactId>test-jetty-ee9-osgi</artifactId>
```

Comment:
Test artifact? I'd remove it.

Reply:
I'll also flag it as `maven.deploy.skip=true`